app/controllers/v1/sessions_controller.rb

```diff
@@ -133,6 +123,7 @@ def render_login(type, previous_saml_uuid = nil)
                                locals: { url: login_url, params: post_params },
                                format: :html
       render body: result, content_type: 'text/html'
+      Rails.logger.info('SSOe SAML sent')
```

Comment:
```suggestion
      Rails.logger.info('SSOe: SAML Request')
```

Reply:
I'd like to also be logging authnContext of both the request and response, but both of those are more involved and could be a separate PR. 

For request, I think it would mean also returning a separate authnContext from the helper methods, or else logging that info directly in the helper, since otherwise it's buried in the form parameters.  

For response, we can log it in the callback_stats method, or anywhere after we've parsed the response, but then we lose the benefit of logging right at the top of the saml_callback method.

Reply:
Just trying to. get apple-apples comparisons. SAML Request/responses can always be in terms of authnContexts; logical logins can be in terms of the "type" from the `new` request.

Reply:
I think thats worth doing now.  For the response I think its ok to do it after parsing, we always have the nginx `/v1/sessions/callback` records to compare it to

Reply:
@patrickvinograd does this work https://github.com/department-of-veterans-affairs/vets-api/pull/4462/commits/5c072e7250dc67e71209e947e3d81ef223ef84a1